edgelet/edgelet-http/src/lib.rs

```diff
@@ -217,9 +223,37 @@ where
 
         let protocol = Arc::new(protocol);
 
+        let (lock, limit) = if let ConcurrencyThrottling::Limited(limit) = concurrency {
+            (Some(Arc::new(AtomicU32::new(*limit))), *limit)
+        } else {
+            (None, 0 as u32)
```

Comment:
nit: you can do `0u32`

Reply:
And I believe clippy will ask you to do `0_u32`

Reply:
Fixed